### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Perpee**
GROUP B
Please review my work @promise-J and @ChidiPatrick and @Pieta9221

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,52 +1,290 @@
...
+    top: 0;
+    left: 0;
+}
+
+header .head {
+    display: flex;
+    align-items: center;
+}
+
+header nav a {
```
Comment (Pieta9221) [inline css/style.css:162]: Try implementing different font-family in your work an do not stick to the default font-family

File: css/style.css [hunk 2]
```diff
@@ -1,52 +1,290 @@
...
+    margin: 20px 0;
+    line-height: 1.5;
+    padding-bottom: 40px;
+}
+
+aside {
+    border-top: solid 1px rgb(156 128 128);
+}
+
+aside div:nth-of-type(1) a {
```
Comment (Pieta9221) [inline css/style.css:228]: Kindly implement this: Use floats to create a two-column layout for the main content area, with one column taking up 70% of the width and the other taking up 30% of the width.

**Review by Pieta9221 — COMMENTED** [review]
Good job
:+1: :+1: :+1:
Kindly read up CSS floats, and implement it, before merging

**Comment by Perpee** [comment]
I used float buh it didn't work
It worked initially buh after i started with mobile view which usually has
one column nd then i tried implementing float in d media query fr desktop
view.. it didn't come out well

**Comment by Pieta9221** [comment]
I don't seem to understand what you are saying... Float should be for the desktop view but it isn't. You can ask your colleagues too how they fixed theirs and if it's still unclear, you can reach out to us in person

**Pull request merged**